openmetadata-ui/src/main/resources/ui/src/components/common/UserTag/UserTag.component.tsx

```diff
@@ -58,12 +58,7 @@ export const UserTag = ({
       )}
       data-testid="user-tag"
       size={8}>
-      <ProfilePicture
-        id={id}
-        name={name}
-        type="circle"
-        width={toString(width[size])}
-      />
+      <ProfilePicture name={id} type="circle" width={toString(width[size])} />
```

Comment:
Please take care of this in other places.

```suggestion
      <ProfilePicture name={id}  width={toString(width[size])} />
```